frontend/src/components/core/Block/ElementNodeRenderer.tsx

```diff
@@ -233,7 +233,9 @@ const RawElementNodeRenderer = (
       )
 
     case "arrowTable":
-      return <ArrowTable element={node.quiverElement as Quiver} />
+      return (
+        <ArrowTable element={node.quiverElement as Quiver} width={width} />
```

Comment:
All FullScreenWrapper-wrapped components require a `width` property, so this was technically a type error before.